docs/usage/offline.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+title: Offline Mode
+permalink: /usage/offline/
+parent: Usage
+nav_order: 4
+---
+
+## Offline mode
+
+{: .note }
+Features and flags with the `experimental` prefix might change or be removed with only a minor version update.
+
+OSV-Scanner now supports offline scanning as an experimental feature. Offline scanning checks your project against a local database instead of calling the OSV.dev API.
+
+### Local Database Option
+
+The local database flag `--experimental-local-db` causes OSV-Scanner to download or update your local database and then scan your project against it. `--experimental-local-db-path` determines the location of the database. 
```

Comment:
`--experimental-local-db-path` is an internal flag so should not be documented

Reply:
So the user won't need to set it?

Reply:
Correct - it's available for the library (which isn't documented yet) but not for the CLI; that stores the databases in the user's cache directory within a directory named `osv-scanner`.

The flag ended up still existing because otherwise tests can't be run in parallel easily, but it's marked as hidden so it'll never show up in the help output and will be considered internal-only.

Btw I'll share the design doc with you to give you a bit more context

Reply:
Thanks @G-Rath